### PR TITLE
Bug fixed issue #22552

### PR DESCRIPTION
**Pull request by shitalmule04**
##### SUMMARY

Bug fixed from file ec2_ami_copy file from Issue [#22552](https://github.com/ansible/ansible/issues/22552)

##### ISSUE TYPE

 - Bugfix Pull Request
 

##### COMPONENT NAME

ec2_ami_copy
##### ANSIBLE VERSION

```
2.5
```

##### ADDITIONAL INFORMATION

 Worked on Minimum permissions required for use of Amazon cloud modules, and improve ansible error output [#22552](https://github.com/ansible/ansible/issues/22552)

```

```

### CODE REVIEW

File: test/sanity/pep8/legacy-files.txt [hunk 1]
```diff
@@ -1,16 +1,16 @@
-lib/ansible/cli/config.py
+lib/ansible/constants.py
```
Comment (Akasurde) [inline test/sanity/pep8/legacy-files.txt]: why add constants.py again if it is available on line number 4 ?

File: test/sanity/pep8/legacy-files.txt [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 lib/ansible/config/data.py
 lib/ansible/config/manager.py
 lib/ansible/constants.py
 lib/ansible/modules/cloud/amazon/_ec2_ami_search.py
 lib/ansible/modules/cloud/amazon/_ec2_remote_facts.py
 lib/ansible/modules/cloud/amazon/_ec2_vpc.py
 lib/ansible/modules/cloud/amazon/_ec2_vpc_dhcp_options.py
-lib/ansible/modules/cloud/amazon/aws_kms.py
+lib/ansible/modules/cloud/openstack/_os_server_actions.py
```
Comment (Akasurde) [inline test/sanity/pep8/legacy-files.txt]: Do not touch file which are already removed.

**Review by Akasurde — CHANGES_REQUESTED** [review]

**Comment by shitalmule04** [comment]
@Akasurde while rebase the branching, after prompting

>$ git pull --rebase upstream devel



  command  I got [this](https://github.com/shitalmule04/issues-screenshot/blob/master/Rebase11.png) output. Please suggest any solution.

**Comment by Akasurde** [comment]
@shitalmule04 There is merge conflict in legacy-files.txt. You can resolve merge conflict by following [this](https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/).

**Comment by s-hertel** [comment]
@shitalmule04 Pull requests that are just fixing pep8 can cause conflicts for other pull requests. We usually prefer to add pep8 fixes to pull requests that fix bugs or add a feature. Pep8 for ec2_ami_copy will be fixed by #24767, so you can remove your changes for that file from this pull request. This doesn't really improve determination of the minimum permissions for the module to run since it's just pep8 fixes. If you have questions feel free to check out the #ansible-aws IRC channel (my handle is shertel).

**Comment by gundalow** [comment]
As agreed in 2017-12-07 Core meeting, we've done bulk pep8 fixes across all modules in `devel`.

This was done in https://github.com/ansible/ansible/pull/33682 and https://github.com/ansible/ansible/pull/33690 which have been merged into `devel`. because of this you will need to rebase your PR. Thanks in advance.

**Comment by gundalow** [comment]
Looking at this PR where is the actual bug fix, all I see are pep8 changed.

If there is no bug fix in here please close this PR

**Pull request closed without merging**